hybrid/replicator/source-topic.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: platform.confluent.io/v1beta1
```

Comment:
@rohit2b We might not need KafkaRestClass here, since we do not have rbac enabled and we do not need to provide kafka rest authentication. And for kafka topic, if kafkaRestClassRef is not provided, it won't go to use default kafkaRestClass right now, may improve this after we deprecated kafkaClusterRef and kafkaRestRef in topic CR.